templates/format-drives.j2

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```

Comment:
I am unclear where this files gets copied over, am I missing something or is there a missing piece to actually put this template in place?

Reply:
@vaubarth I'm sorry I forgot to add the renamed folder `vmimage`, I just added back and you will see easily where it's copied (`tasks/vmimage/main.yml`)